src/selectors/entities/users.ts

```diff
@@ -384,10 +383,22 @@ export function searchProfiles(state: GlobalState, term: string, skipCurrent = f
     return filteredProfiles;
 }
 
+export function searchProfilesStartingWithTerm(state: GlobalState, term: string, skipCurrent = false, filters?: Filters): Array<UserProfile> {
+    const users = getUsers(state);
+    const profiles = filterProfilesStartingWithTerm(Object.keys(users).map((key) => users[key]), term);
```

Comment:
Could `Object.keys(users).map((key) => users[key])` be replaced with `Object.values(users)` instead?

Reply:
Sure